integration-test/src/main/resources/application.yml

```diff
@@ -333,6 +342,8 @@ integrationtest:
         instanceProfile: "arn:aws:iam::1234567890:instance-profile/mock.testing.instance.profile"
       baseLocation: "s3a://mock-test"
       fileSystemType: S3
+      dataAccessRole:
```

Comment:
we already added this to e2e tests on jenkins right @aszegedi ?

Reply:
What we really-really need here is the:
```
idbmms:
    # Here we need a valid idbroker mapping management host (Manowar Dev)!
    host:
```